analyses/org.eclipse.tracecompass.incubator.executioncomparision.core/src/org/eclipse/tracecompass/incubator/internal/executioncomparision/core/DifferentialSeqCallGraphAnalysis.java

```diff
@@ -360,7 +368,16 @@ public void dispose() {
      */
     private void setDifferentialCallGraphProvider(DifferentialCallGraphProvider differentialCallGraphProvider) {
         fDifferentialCallGraphProvider = Objects.requireNonNull(differentialCallGraphProvider);
-
+    }
+    private static ITmfTrace getTrace(String traceName) {
```

Comment:
this needs to be annotated <at>nullable guess what, in github, nullable is a user that gets pinged!! @PatrickTasse

Reply:
I can't see why, its called in this loop: for (String traceName : traceList) { ITmfTrace trace = getTrace(traceName);} and it can not be null.